Patches/RH2 Rimmu-Nation² - Weapons/RM2_CE_Pistols.xml

```diff
@@ -348,7 +348,7 @@
 					<Properties>
 						<verbClass>CombatExtended.Verb_ShootCE</verbClass>
 						<hasStandardCommand>true</hasStandardCommand>
-						<defaultProjectile>Bullet_45ACP_FMJ</defaultProjectile>
```

Comment:
This incorrect, `ammoSet` does not belong in `properties`.

Reply:
I think it was left over from the original patches; I just added the new weapons to the bottom of the patch file, but I will try to fix it up and count it as exercise.

Reply:
Yep, looks correct now.